sumo/cli/bandplot.py

```diff
@@ -750,7 +759,10 @@ def _get_parser():
         "--orbitals",
         type=_el_orb,
         metavar="O",
-        help=("orbitals to split into lm-decomposed " 'contributions (e.g. "Ru.d")'),
+        help=(
+            "orbitals to split into lm-decomposed "
+            'contributions (e.g. "Ru.d")'
+        ),
```

Comment:
I don't understand what is going on with the linting here.

Obviously a human would never have written the existing version. Did some other auto-linter remove the line-break? And if so, did it break PEP8 in the process? @utf any idea?

This is why I dislike auto-formatters...

Reply:
Checked the blame, yes `black` did this.

Reply:
I checked the blame, and indeed this was ruined by `black`. The original version was

```python
    parser.add_argument('--orbitals', type=_el_orb, metavar='O',
                        help=('orbitals to split into lm-decomposed '
                              'contributions (e.g. "Ru.d")'))
```